exchange/howto/requirements-for-adding-an-altcoin.adoc

```diff
@@ -0,0 +1,66 @@
+= Requirements for making a request for adding an Altcoin or token
+:toc:
+
+== Introduction
+
+This document describes the process how to submit a request for adding a new Altcoin or token to the currency list in Bisq. +
+Adding a coin/token does not mean any endorsement from the Bisq community. For making decisions in case of controversial requests we use the Bisq DAO voting process. +
+Please don't send promotional/background information about the coin/token but only fulfill the formal requirements below.
+
+Please note that Bisq is not implementing technically any coin/token infrastructure (wallet, full nodes, block explorer,...) but only supports that the users can add the altoin address to a payment account and then during the trade process the users exchange the Altcoin address. The process of sending and receiving the Altcoin is not part of our system (users use their own Altcoin wallets).
+
+Already added coins/tokens which have been traded at least once or which have provided a PR for the address validation will be kept. All other coins/tokens will be removed when we start introducing that requirements. +
+This document replaces the process described in the Bisq forum [1].
+
+== Process
```

Comment:
I suggest that all HOWTO docs should have an **Introduction** section (as you already have above), and an **Instructions** section, as seen in https://github.com/bisq-network/docs/blob/master/exchange/howto/manual-dispute-payout.adoc.

In the case of this document, we should probably have a **Prerequisites** section as well: a section that tells the reader what they're going to need to have in order to walk through the steps in the instructions that follow. For example, the token they are trying to get listed must already have a block explorer. That is a prerequisite for following the instructions that follow, not a step in and of itself. 

In any case, once the reader gets to the **Instructions** section, it should be as simple and straightforward as possible: Step 1: Do X; Step 2: Do Y, and so forth. The idea with HOWTO docs is that by their nature, they provide readers with extremely clear, fool-proof, permalink-friendly, step-by-step instructions that minimize the need for re-explaining stuff on our side and/or rework on the reader's side. (By permalink-friendly, I mean that each step should have its own heading so that we can point users directly to the exact step(s) they forgot or otherwise failed to complete with an anchor link, e.g. https://github.com/bisq-network/docs/blob/master/exchange/howto/manual-dispute-payout.adoc#step-3-send-your-private-key-to-your-arbitrator.

So for this doc, I'm picturing a structure like the following:

 - Introduction
 - Prerequisites
 - Instructions
   - Step 1: ...
   - Step 2: ...
 - Caveats
   - Controversial tokens may be put to a vote
   - Inactive tokens will be de-listed